torch/csrc/jit/tracer.h

```diff
@@ -233,4 +253,24 @@ inline bool VariableFlags::verify(const Variable& var) {
   return !was_null && requires_grad == var.requires_grad() && is_volatile == var.is_volatile();
 }
 
+Node* recordTraceWorker(std::string op, at::ArrayRef<Variable> inputs, at::ArrayRef<Variable> outputs);
```

Comment:
nit: when I see `worker` I assume it's going to be a loop for a background thread. But maybe it's just me. Why not make it another `recordTrace` overload?

Reply:
How about Helper? The overload is asymmetric so I want to make it more clear.

Reply:
asymmetric in what sense? Helper is fine